12 MolecularGlassOrientationMapping.ipynb

```diff
@@ -0,0 +1,1038 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.    axs.quiver(nav_axes[1].axis[::sep],nav_axes[0].axis[::sep], vx[::sep,::sep], vy[::sep, ::sep],)
</pre><p>Perhaps <strong>axs.streamplot</strong> could also be plotted as well as quiver. I think streamplots looks a better for orientation maps personally.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyxem/pyxem-demos/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>